src/components/home/latest.tsx

```diff
@@ -12,18 +12,18 @@ const Latest = ({ title, data }: LatestProps) => {
 
```

Comment:
Capitalize

Reply:
I believe it is not capitalizing because it is the same name. I'll create another pr